src/Data/List/Relation/Binary/Sublist/Setoid/Properties.agda

```diff
@@ -203,6 +205,31 @@ module _ {as bs : List A} where
   reverse⁻ : reverse as ⊆ reverse bs → as ⊆ bs
   reverse⁻ = HeteroProperties.reverse⁻
 
+------------------------------------------------------------------------
+-- merge
+
+module Merge {ℓ′} (_≤_ : Rel A ℓ′) (dto : IsDecTotalOrder _≈_ _≤_)  where
+
+  open IsDecTotalOrder dto using (_≤?_)
+
+  merge-is-superlistˡ : ∀ xs ys → xs ⊆ merge _≤?_ xs ys
```

Comment:
I think the standard naming convention would be dictate either `⊆-mergeˡ` or `xs⊆merge[xs,ys]`?

Reply:
Thanks, I'll fix this.